torchgeo/models/croma.py

```diff
@@ -123,17 +123,16 @@ def forward(
         """
         return_dict: dict[str, Tensor] = {}
 
-        device = x_sar.device if x_sar is not None else x_optical.device
-        attn_bias = self.attn_bias.to(device)
```

Comment:
@nilsleh check if this is valid. I think if the model is transferred to a device, so will all of its attributes, so this line isn't needed. This line doesn't work in the case where both x_sar and x_optical are None, and adding:
```python
assert x_sar or x_optical
```
wasn't sufficient to convince mypy.

Reply:
will have to test this live and potentially open another PR to fix it.